bazel/utils/files.bzl

```diff
@@ -143,7 +143,9 @@ def files_to_dir(ctx, dirname, paths, post = ""):
     roots = {}
     for f in paths:
       root = f.path
-      if f.owner and f.owner.workspace_root:
+      # Check for external/ prefix because there are files that belong to
```

Comment:
do you have an example of those prefixes?

Reply:
Added the motivation in the new PR: https://github.com/enfabrica/enkit/pull/607